### DISCUSSION TITLE
Commits que não exibem no github (mesmo em repos pulicos)

### DISCUSSION THREAD

**Discussion by YuriLRodrigues** [community/community#75626] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Estou passando um certo problema, mesmo que eu dê commits em um projeto público, ele mesmo assim não está aumentando a minha contagem de commits, alguém poderia me ajudar ?

**Comment by santoshvandari** [comment]
Aqui estão as condições que devem ser atendidas para contar seu comprometimento:

- O commit deve ser feito dentro de 24 horas.
- Seu email de configuração local do git deve ser o mesmo que o da sua conta do GitHub ou precisa conectar esse email à sua conta do GitHub.
- O commit deve ser feito no branch padrão ou deve ser posteriormente mesclado ao branch padrão.
- Commits em repositórios bifurcados não são contados. Você precisa abrir um PR para o repositório pai e ele deve ser mesclado.

Se todas essas condições forem cumpridas, você pode entrar em contato com a equipe de suporte do GitHub.

**Reply by YuriLRodrigues** [reply]
onde eu posso ver se minha conta está vinculada ? porque eu passei o git config user.email certinho e password tbm, mas msm assim n ta funcionando, tem alguma caminho para seguir ?

**Comment by santoshvandari** [comment]
Você só precisa garantir que o email de configuração do git seja o mesmo que o email do GitHub no seu perfil. Verifique seu email em Configurações > Email.

**Reply by YuriLRodrigues** [reply]
está tudo certo, mas mesmo assim não esta contabilizando

**Comment by queenofcorgis** [comment]
Hi @YuriLRodrigues , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
